app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -164,18 +215,55 @@ end
 
 ## Demonstration
 
-1. Create a Service and a Route in Kong:
+### Step 1: Create a Service in Kong
+
+   {% navtabs %}
+   {% navtab Using cURL %}
+
+   ```bash
+   curl -i -X POST http://<admin-hostname>:8001/services \
+    --data name=example.com \
+    --data url='http://mockbin.org'
+   ```
+
+   {% endnavtab %}
+   {% navtab Using HTTPie %}
+
+   ```bash
+   $ http :8001/services name=example.com host=mockbin.org
+   ```
+
+   {% endnavtab %}
+   {% endnavtabs %}
+
+### Step 2: Create a Route in Kong
 
-    ```bash
-    $ http :8001/services name=example.com host=mockbin.org
-    $ http -f :8001/services/example.com/routes hosts=example.com
-    ```
+   {% navtabs %}
+   {% navtab Using cURL %}
 
-2. Create a file named `transform.lua` with the transformation code. The
-   following example adds a header, appends "arr!" to any message, and adds
-   an `error` and `status` field on the response.
+```bash
+curl -i -X POST http://<admin-hostname>:8001/services/example.com/routes \
+  --data 'paths[]=/mock' \
+  --data name=mocking
```

Comment:
To align this example with the other steps or the httpie command, it should add the route as a host and not a path

```suggestion
  --data 'hosts[]=example.com'
```